internal/flickr/flickr.go

```diff
@@ -0,0 +1,146 @@
+package flickr
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/anz-bank/sysl-go/common"
+	"github.com/anz-bank/sysl-go/config"
+	"github.com/anz-bank/sysl-go/core"
+	"github.com/anz-bank/sysl-go/handlerinitialiser"
+	"github.com/anz-bank/sysl-template/internal/flickr/handlers"
+	"github.com/anz-bank/sysl-template/internal/gen/flickr"
+	"github.com/go-chi/chi"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// ListenAndServe func
+func ListenAndServe(cfg *ConfigContainer) error {
```

Comment:
Nit: Can you describe what these funcs actually do?

Reply:
ok